src/model/TaskState.model.js

```diff
@@ -21,8 +21,6 @@ import { Enumify } from 'enumify'
  * Cylc valid task states.
  */
 export class TaskState extends Enumify {
-  // NOTE: the order of the enum values is important to calculate the group states in the UI. Items at
-  // the top of the list have preference over the items below in the algorithm.
```

Comment:
There's an array used to sort things in the UI. Expand the file here or click View File, then scroll down. Which means this comment is outdated.